src/sections/HomeHero/index.module.css

```diff
@@ -0,0 +1,26 @@
+@import "../../styles/settings.css";
+
+.root {
+  position: relative;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: $size-148 $column $size-294 $column;
+
+  background: linear-gradient(94.99deg, #f9f1e5 13.96%, #f9f1e5 90.87%);
+  border-radius: 128px;
+}
+
+.copy > * + * {
```

Comment:
Why? This is slow as hell, it's barely readable and saves just a few `classNames`. Any other reason to use this besides shaving off a few lines of css?

Reply:
In 2020, universal selectors impact is negligible, if so, completely inexistent (unless you apply let's say, a transition or a box-shadow to every element on the page). On Javascript selectors? Yeah really slow. On CSS? Not that much. Close to 0 impact. https://www.telerik.com/blogs/css-tip-star-selector-not-that-bad (2012 study)

I did this just because our components don't have class names and I don't want to turn the website into a div soup just because. I can't style a Text element without wrapping it into a single div

I'll use a `className` if I can feed it into our components, but then again, every single UI component would need a className prop.

Reply:
I think you're missing the point, notice that the alternative I'm thinking is adding a single div either around `Title` or `Subtitle` to add the margin. I don't see how this triggers a so called `div` soup or how you should add `className` as a prop to the other components.

Reply:
What I'm concerned if about readability and ease of other people to understand the code and make changes. I don't have issues with this approach in isolation, but if we start applying this as a pattern then this will grow unnecessarily complex IMO

Reply:
Every `Text`, `Title`, `Subtitle`, `Button` that you want to style need to be wrapped in a `div` just because. If you do this more than 5 times, which you are 100% going to, you get a `div` soup. I just don't like to use `divs` all that much.

But it doesn't matter, too late to go back, I'll put the `div` there